requirements.txt

```diff
@@ -1,4 +1,4 @@
-requests
+requests>=2.4.2
```

Comment:
what's missing in requests bellow 2.4.2

Reply:
parameter "json" for get and post ... you've used data which was is no correctly converted to content type application/json. Therefore I've used "json=" and it was introduced in 2.4.2